var/spack/repos/builtin/packages/xyce/package.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Xyce(CMakePackage):
+    """Xyce (zīs, rhymes with “spice”) is an open source, SPICE-compatible,
+    high-performance analog circuit simulator, capable of solving extremely
+    large circuit problems by supporting large-scale parallel computing
+    platforms. It also supports serial execution on all common desktop platforms,
+    and small-scale parallel runs on Unix-like systems. In addition to analog
+    electronic simulation, Xyce has also been used to investigate more general
+    network systems, such as neural networks and power grids.
+    """
```

Comment:
If you split the docstring in a first short paragraph followed by a longer description, e.g. like:
```suggestion
    """Xyce (zīs, rhymes with “spice”) is an open source, SPICE-compatible,
    high-performance analog circuit simulator, capable of solving extremely
    large circuit problems by supporting large-scale parallel computing
    platforms. 
    
    It also supports serial execution on all common desktop platforms,
    and small-scale parallel runs on Unix-like systems. In addition to analog
    electronic simulation, Xyce has also been used to investigate more general
    network systems, such as neural networks and power grids.
    """
```
Spack will use the short paragraph wherever a brief description is needed and the entire docstring instead in contexts where a longer description fits better.